src/vabackend.c

```diff
@@ -762,6 +819,14 @@ static VAStatus nvCreateContext(
         return VA_STATUS_ERROR_UNSUPPORTED_PROFILE; //TODO not sure this is the correct error
     }
 
+    cudaVideoChromaFormat chromaFormat = cfg->chromaFormat;
+    int bitDepth = cfg->bitDepth;
+    if (num_render_targets) {
+        NVSurface *surface = (NVSurface *) getObjectPtr(drv, render_targets[0]);
+        chromaFormat = surface->format;
+        bitDepth = surface->bitDepth;
```

Comment:
We should probably update the config chroma and bitDepth with the ones we took from the render target, just so that match.